pipeline/metadata/4.3.yaml

```diff
@@ -0,0 +1,534 @@
+# =====================================================================================
+# Metadata file for 4.3 RHCS release version execution for pipeline V3.
+# Single file to execute the all tier-1, tier-2, tier-3 and tier-4 suites in parallel
```

Comment:
This can be changed as "Single file to specify test suites to be executed for all regression (perbuild) and periodical tests"

Reply:
Changed the Doc string as per comments